curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/653283d07b8f9d294aafa83b.md

```diff
@@ -0,0 +1,637 @@
+---
+id: 653283d07b8f9d294aafa83b
+title: Step 20
+challengeType: 0
+dashedName: step-20
+---
+
+# --description--
+
+Define a `playSong` function using `const`. The function should take an `id` parameter which will represent the unique identifier of the song you want to play. 
+
+The `playSong` function is responsible for playing a selected song in your music player.
+
+# --hints--
+
+Test 1
+
+```js
+
+```
```

Comment:
The tests:

You should use `const` to create an empty function named `playSong`.

```js
assert.match(code, /const\s+playSong\s*=\s*/)
```

`playSong` should be a function.

```js
assert.isFunction(playSong)
```

Your `playSong` function should use an arrow syntax.

```js
assert.match(code, /const\s+playSong\s*=\s*\(.*\)\s*=>\s*/)
```

Your `playSong` function should take an `id` parameter.

```js
assert.match(code, /const\s+playSong\s*=\s*\(id/)
```

Your `playSong` function  should be empty.

```js
assert.match(code, /const\s+playSong\s*=\s*\(id\)\s*=>\s*\{\n?\s*?\};?/)
```